src/main/java/lesson_3/exercises/CoinToss.java

```diff
@@ -0,0 +1,40 @@
+package lesson_3.exercises;
+
+import java.util.Random;
+import java.util.Scanner;
+
```

Comment:
Please add the description of the task as comment in the code

Reply:
added descriptions